datadriven/src/test_data.rs

```diff
@@ -22,7 +22,21 @@ impl fmt::Display for CmdArg {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self.vals.len() {
             0 => write!(f, "{}", self.key),
-            _ => write!(f, "{}={:?}", self.key, self.vals),
+            1 => {
+                let mut str = format!("{}=", self.key);
+                str += &self.vals[0];
+                write!(f, "{}", str)
+            }
+            _ => {
+                let mut str = format!("{}=(", self.key);
```

Comment:
Why not `write!(f, "{}=({})", self.key, self.vals.join(", "))`?

Reply:
Cool!